src/_examples/index.md

```diff
@@ -20,15 +30,15 @@ title: Examples
 
 {% include components/standalone-project.html
    name='iFixit PWA'
-   description='A React-based web app using <code>workbox-build</code> in a <code>gulp</code>-based build process.'
+   description='A React-based web app using workbox-build in a gulp-based build process.'
    img-src='/images/third_party/ifixit.png'
    source='https://github.com/GoogleChrome/application-shell/tree/ifixit-pwa/ifixit-pwa'
    demo='https://ifixit-pwa.appspot.com/'
 %}
 
 {% include components/standalone-project.html
    name='React HN'
-   description='A React-based web app using <code>workbox-cli</code> in an <code>npm scripts</code>-based build process.'
+   description='A React-based web app using workbox-cli in an npm scripts-based build process.'
    img-src='/images/third_party/react-hn.png'
    source='https://github.com/insin/react-hn/tree/sw-helpers'
```

Comment:
Want to update this to point to https://react-hn.appspot.com and https://github.com/insin/react-hn now that it's no longer in branch?